1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +17,12 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+    
+    let i = 0;
+    while (birthdays[i].substring(0, 4) !== "July") {
```

Comment:
This is works and is perfectly fine.
Another way to achieve the same thing here is using the String startsWith method: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith